src/components/Navbar.js

```diff
@@ -38,7 +39,42 @@ const Navbar = () => {
         </p>
         <BsCodeSlash color="#fff" size={33} />
       </div>
+      {localStorage.getItem('username') ?
```

Comment:
You can access it in context wrapper and pass it down through context instead of directly accessing local storage